abstract.txt

```diff
@@ -0,0 +1,14 @@
+Gammapy - A Python package for gamma-ray astronomy
+
+Johannes King, Lea Jouvin, Axel Donath, Christoph Deil, Dirk Lennarz, Ellis
+Owen, Jonathan Harris, Luigi Tibaldo, Manuel Paz Arribas, Olga Vorokh, Regis
+Terrier, Rolf Bühler, Stefan Klepser, Victor Zabalza, Brigitta Sipocz
+
+Gammapy is an open-source Python package for gamma-ray astronomy. It is an
+in-development affiliated package of Astropy that builds on the core scientific
+Python stack (Numpy, Scipy, Astropy) to provide tools to simulate and analyse
```

Comment:
For simulations there's [fake](http://cxc.harvard.edu/sherpa/ahelp/fake.html) in Sherpa which is basically `np.poisson` of `npred`, but still a very powerful simulation tool. We need to add more and make simulation a priority for the coming months, it's very important for code and method verification and obviously for CTA.
So I'd like to keep this aspect in the Gammapy description.